Ginger/GingerCore/Platforms/PlatformsInfo/PlatformInfoBase.cs

```diff
@@ -53,6 +53,10 @@ public abstract class PlatformInfoBase: IPlatformInfo
         public abstract List<ActUIElement.eElementDragDropType> GetPlatformDragDropTypeList();
 
         public abstract ObservableList<ElementLocator> GetLearningLocators();
+
+        public abstract string GetPageUrlRadioLabelText();
```

Comment:
i think these methods needs to be virtual and by default the option will be URL and Go to Page.
And only for relevant platforms we will override it